clients/osv/src/main/kotlin/Model.kt

```diff
@@ -58,7 +58,14 @@ data class Vulnerability(
     val related: List<String> = emptyList(),
     val summary: String? = null,
     val details: String? = null,
+
+    /**
+     * Optionally multiple representations (with different scoring algorithms) of the severity associated with this
```

Comment:
Sorry, more nits, I didn't really this fully before:
- Comma after "Optionally"
- Something is missing in this sentence, maybe an "are" before "associated"?

Reply:
I added the comma, but adding an "are" changed the meaning of the sentence to something which doesn't make sense to me. So I ended up re-writing it again.